tests/integration_tests/tests/test_cases.py

```diff
@@ -91,9 +90,6 @@ def copy_file_to_manager(self, source, target, owner=None):
                 self.env.container_id,
                 ['chown', owner, target]
             )
-        do_retries_boolean(docker.file_exists,
-                           container_id=self.env.container_id,
-                           file_path=target)
```

Comment:
I've been trying to figure out why this was even introduced, thought there might be some sort of race condition during the copy, but it doesn't look like that was ever commented, so... I guess we'll find out.

Reply:
Perhaps there was, back when this was using ssh